dev/PushNotifications/PushNotificationsLongRunningTask/NotificationListener.cpp

```diff
@@ -28,3 +32,17 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationListener::OnRawNotificationReceived
     return S_OK;
 }
 CATCH_RETURN()
+
+STDMETHODIMP_(HRESULT __stdcall) NotificationListener::OnToastNotificationReceived(
+    ABI::Microsoft::Internal::ToastNotifications::INotificationProperties* notificationProperties,
```

Comment:
Nit: you can do `using namespace` to shorten these lengthy ABI namespaces in the code.